docs/README.md

```diff
@@ -125,8 +125,9 @@ The component rendered when used the `checkout-summary.compact` block.
 | Prop name | Type | Required |
 | --- | --- | --- |
 | [`totalizers`](#summarysmall-totalizers) | `array` | `true` |
-| [`totalizersToShow`](#summarysmall-totalizerstoshow) | `array` | `true` |
+| [`totalizersToShow`](#summarysmall-totalizersToShow) | `array` | `true` |
 | [`total`](#summarysmall-total) | `number` | `true` |
+| [`totalCalculation`](#summarysmall-totalCalculation) | `enum` | `visibleTotalizers` |
```

Comment:
Same here. Also, I think you meant to put `visibleTotalizers` as a default value and not as a required explanation, right?

Reply:
Yep 😂 
This README is a bit different from our default template, I got confused

Reply:
I think I'll remove this `Required` column in favor of a `default` one... we can signal props as required using that little red badge we use in some of our docs

Reply:
Yes, I think that will be good :)